src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -289,13 +281,6 @@ function InteractiveMediaEditor({ content, onContentChanged }) {
           </Fragment>
         )}
       </Form>
-
-      {isDeterminingDuration && (
```

Comment:
We could still show this when `mediaDuration.isResolved === false`

Reply:
That's true. However do you think it's necessary? In most cases it would just be a very quick flash on the screen that the user wouldn't be able to read.
If the duration determination takes too long, nothing is actually broken in the UI anymore, everything is usable still and shown in percentages.
Thought?

Reply:
Necessary, no. Just if we want the exact old behavior. But I think we do not indicate the loading in other components, so we could drop it here, too.

Reply:
👍